data/data-pipeline/data_pipeline/etl/base.py

```diff
@@ -77,8 +91,51 @@ class ExtractTransformLoad:
     #  periods. https://github.com/usds/justice40-tool/issues/964
     EXPECTED_MAX_CENSUS_TRACTS: int = 74160
 
+    # We use ourput_df as the final dataframe to use to write to the CSV
+    # It is used on the "load" base class method
     output_df: pd.DataFrame = None
 
+    @classmethod
+    def yaml_config_load(cls) -> dict:
+        # check if the class instance has score YAML definitions
+        datasets_config = load_yaml_dict_from_file(
+            cls.DATASET_CONFIG / "datasets.yml",
+            DatasetsConfig,
+        )
+
+        # get the config for this dataset
+        try:
+            dataset_config = next(
+                item
+                for item in datasets_config.get("datasets")
+                if item["module_name"] == cls.NAME
+            )
+        except StopIteration:
+            # Note: it'd be nice to log the name of the dataframe, but that's not accessible in this scope.
+            logger.error(
+                f"Exception encountered while extracting dataset config for dataset {cls.NAME}"
+            )
+            sys.exit()
+
+        # set some of the basic fields
+        cls.LAST_UPDATED_YEAR = dataset_config["last_updated_year"]
```

Comment:
This is interesting. I think we could go back to the `dataset_year` folder structure, but we still need to think through column names. I think this is actually going to be a bigger headache than we can easily address?

Reply:
I think we should kick this can to the versioning discussion and continue on as is right now.

Reply:
Hmm. We can kick the can but we also know this will definitely be a problem for v1.0. Definition M (CEJST Beta) uses older EJScreen data, and we're planning to update the EJScreen data year for Narwhal. Without being able to maintain two years at once, we will either not be able to accurately recreate Definition M or we will not be able to accurately create Narwhal.